doc/source/tune/guides/plot_tune-usage.rst

```diff
@@ -0,0 +1,375 @@
+
+Then, after you run a experiment, you can visualize your experiment with TensorBoard by specifying the output directory of your results.
+
+.. code-block:: bash
+
+    $ tensorboard --logdir=~/ray_results/my_experiment
+
+If you are running Ray on a remote multi-user cluster where you do not have sudo access, you can run the following commands to make sure tensorboard is able to write to the tmp directory:
+
+.. code-block:: bash
+
+    $ export TMPDIR=/tmp/$USER; mkdir -p $TMPDIR; tensorboard --logdir=~/ray_results
+
+.. image:: ../../ray-tune-tensorboard.png
+
+If using TF2, Tune also automatically generates TensorBoard HParams output, as shown below:
+
+.. code-block:: python
+
+    tune.run(
+        ...,
+        config={
+            "lr": tune.grid_search([1e-5, 1e-4]),
+            "momentum": tune.grid_search([0, 0.9])
+        }
+    )
+
+.. image:: ../../images/tune-hparams.png
+
+Console Output
+--------------
+
+The following fields will automatically show up on the console output, if provided:
+
+1. ``episode_reward_mean``
+2. ``mean_loss``
+3. ``mean_accuracy``
+4. ``timesteps_this_iter`` (aggregated into ``timesteps_total``).
+
+Below is an example of the consoe output:
+
+.. code-block:: bash
+
+    == Status ==
+    Memory usage on this node: 11.4/16.0 GiB
+    Using FIFO scheduling algorithm.
+    Resources requested: 4/12 CPUs, 0/0 GPUs, 0.0/3.17 GiB heap, 0.0/1.07 GiB objects
+    Result logdir: /Users/foo/ray_results/myexp
+    Number of trials: 4 (4 RUNNING)
+    +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
+    | Trial name           | status   | loc                 |    param1 | param2 |    acc | total time (s) |  iter |
+    |----------------------+----------+---------------------+-----------+--------+--------+----------------+-------|
+    | MyTrainable_a826033a | RUNNING  | 10.234.98.164:31115 | 0.303706  | 0.0761 | 0.1289 |        7.54952 |    15 |
+    | MyTrainable_a8263fc6 | RUNNING  | 10.234.98.164:31117 | 0.929276  | 0.158  | 0.4865 |        7.0501  |    14 |
+    | MyTrainable_a8267914 | RUNNING  | 10.234.98.164:31111 | 0.068426  | 0.0319 | 0.9585 |        7.0477  |    14 |
+    | MyTrainable_a826b7bc | RUNNING  | 10.234.98.164:31112 | 0.729127  | 0.0748 | 0.1797 |        7.05715 |    14 |
+    +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
+
+You can use a :ref:`Reporter <tune-reporter-doc>` object to customize the console output.
+
+
+Uploading Results
+-----------------
+
+If an upload directory is provided, Tune will automatically sync results to the given directory, natively supporting standard S3/gsutil commands.
+
+You can customize this to specify arbitrary storages with the ``sync_to_cloud`` argument. This argument is similar to ``sync_to_cloud`` in that it supports strings with the same replacement fields and arbitrary functions.
+
+.. code-block:: python
```

Comment:
what would this function look like?  what does the function take in?

Reply:
GREAT CATCH i'll have to fix this